docs/website/config.toml

```diff
@@ -0,0 +1,17 @@
+# The URL the site will be built for
+base_url = "https://bytesnake.github.io/linfa/"
```

Comment:
Do we want this to be hosted at bytesnake.github.io, or create a rust-ml.github.io page?

Reply:
no I forgot to update the link, this will be only valid after merging the PR